application/comit_node/src/swap_protocols/ledger/mod.rs

```diff
@@ -51,3 +47,11 @@ pub trait Ledger:
 
     fn address_for_identity(&self, identity: Self::Identity) -> Self::Address;
 }
+
+#[derive(Clone, Derivative)]
+#[derivative(Debug = "transparent")]
```

Comment:
fancy 🎩

Reply:
what this do?

Reply:
I derives Debug as if the type would not be wrapped.